libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -404,6 +404,24 @@ enum BuyBitcoinProvider {
     "Moonpay",
 };
 
+[Enum]
+interface PrepareWithdrawExtra {
+    UnexpectedFunds();
```

Comment:
I don't understand this extra enum. We only have one withdraw type. I don't think it is needed.

Reply:
I'll remove it because we are not doing the "spontaneous transaction" with off chain balance. That enum was an abstraction to it, but I believe it will not be necessary after our slack chat.